v2/emailpassword/custom-ui/init/user-management-dashboard/tenant-management/details.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: details
+title: "Tenant Details"
+hide_title: true
```

Comment:
not directly related to this file, however a few points:
- In the code tabs where you show the dashboard option, it needs to be the first option and the default one thats selected. So it would be Dashboard, NodeJs, ... 
- In SAML integration docs, the flow is broken cause in step 3, you say send xml to boxy, and then in step 6, we add the xml to the dashboard as well again, in which case, step 3 is not needed. To solve this, you can break the page into two clear sections -> Using dashboard, Using API. And then talk about each flow separately from scratch. You can even make this into two pages (Using dashboard, Using API) and put them under the integration section (you will have to make the integration page into a section)
- There are places in the docs which go like "To configure otp-email requirement for a tenant, we can call the following API:", and then the subsequent section has the dashboard UI. For example, in here: https://deploy-preview-829--admiring-bhabha-7b1be9.netlify.app/docs/mfa/email-sms-otp/otp-for-all-users. Now this can be solved by changing the text before like "To configure otp-email requirement for a tenant, we can call the following API / do the following action on the dashboard".

Reply:
Done